Wonder what went wrong here;

```
Stacktrace
tests/integration/api_build_test.py:16: in <module>
    class BuildTest(BaseAPIIntegrationTest):
tests/integration/api_build_test.py:164: in BuildTest
    @requires_api_version('1.22')
tests/helpers.py:56: in requires_api_version
    docker.utils.version_lt(test_version, version),
docker/utils/utils.py:70: in version_lt
    return compare_version(v1, v2) > 0
docker/utils/utils.py:61: in compare_version
    if s1 == s2:
/usr/local/lib/python3.7/distutils/version.py:46: in __eq__
    c = self._cmp(other)
/usr/local/lib/python3.7/distutils/version.py:170: in _cmp
    if self.version != other.version:
E   AttributeError: 'StrictVersion' object has no attribute 'version'
```

Python 3.7 / 3.8 issue? I saw https://community.letsencrypt.org/t/certbot-failing-to-find-python-version-on-centos-6-10/105634/7, which also seemed to related to Python version compatibility

--- a/hack/make/test-docker-py
+++ b/hack/make/test-docker-py
@@ -7,20 +7,15 @@ source hack/make/.integration-test-helpers
 # TODO docker 17.06 cli client used in CI fails to build using a sha;
 # unable to prepare context: unable to 'git clone' to temporary context directory: error fetching: error: no such remote ref ead0bb9e08c13dd3d1712759491eee06bf5a5602
 #: exit status 128
-: "${DOCKER_PY_COMMIT:=4.2.2}"
-
-# Override the API version used to make sure all tests are run
-# TODO remove this or unset this after https://github.com/docker/docker-py/pull/2512 is merged
-: "${DOCKER_PY_TEST_API_VERSION:=1.39}"
+: "${DOCKER_PY_COMMIT:=4.3.0}"
 
 # custom options to pass py.test
-# TODO remove these skip once we update to a docker-py version that has https://github.com/docker/docker-py/pull/2624: CreateContainerTest::test_invalid_log_driver_raises_exception
 #
 # This option can be used to temporarily skip flaky tests (using the `--deselect`
 # flag) until they are fixed upstream. For example:
 # --deselect=tests/integration/api_container_test.py::AttachContainerTest::test_attach_no_stream
 # TODO re-enable test after https://github.com/docker/docker-py/issues/2513 has been resolved
-: "${PY_TEST_OPTIONS:=--junitxml=${DEST}/junit-report.xml --deselect=tests/integration/api_container_test.py::AttachContainerTest::test_attach_no_stream --deselect=tests/integration/api_container_test.py::CreateContainerTest::test_invalid_log_driver_raises_exception}"
+: "${PY_TEST_OPTIONS:=--junitxml=${DEST}/junit-report.xml --deselect=tests/integration/api_container_test.py::AttachContainerTest::test_attach_no_stream}"
 (
 	bundle .integration-daemon-start
 
@@ -61,7 +56,7 @@ source hack/make/.integration-test-helpers
 	(
 		[ -n "${TESTDEBUG}" ] && set -x
 		# shellcheck disable=SC2086,SC2140
-		exec docker run --rm ${run_opts} -e DOCKER_TEST_API_VERSION="${DOCKER_PY_TEST_API_VERSION}" --mount type=bind,"src=${ABS_DEST}","dst=/src/${DEST}" "${docker_py_image}" pytest ${PY_TEST_OPTIONS} tests/integration
+		exec docker run --rm ${run_opts} --mount type=bind,"src=${ABS_DEST}","dst=/src/${DEST}" "${docker_py_image}" pytest ${PY_TEST_OPTIONS} tests/integration
 	)
 	bundle .integration-daemon-stop
 ) 2>&1 | tee -a "$DEST/test.log"

